lizmap/modules/lizmap/lib/Request/WMSRequest.php

```diff
@@ -188,7 +187,8 @@ protected function getcontext()
             array('repository' => $this->repository->getKey(), 'project' => $this->project->getKey())
         );
         $sUrl = str_replace('&', '&amp;', $sUrl).'&amp;';
-        $data = preg_replace('/xlink\:href=".*"/', 'xlink:href="'.$sUrl.'&amp;"', $response->data);
+        $data = $response->data;
```

Comment:
why this change?

Reply:
https://github.com/3liz/lizmap-web-client/pull/1943#pullrequestreview-541093390